.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "es2015"
```

Comment:
I don't understand babel presets. Isn't it recommended to not use a year, and instead use `env`?

Reply:
Yeah, I keep track of these things. But I'll env instead